app/views/workflows/edit.html.erb

```diff
@@ -78,7 +77,6 @@ See doc/COPYRIGHT.rdoc for more details.
           <%= render partial: 'form',
                      locals: { name: 'author', workflows: @workflows['author'] } %>
         </div>
-      <%= javascript_tag "hideFieldset($('author_workflows'))" unless @workflows['author'].present? %>
```

Comment:
Now the fieldsets are always displayed. Is this intentional? Otherwise, we would have to add the `collapsed` class based on the removed condition.